server/collection/queries.ts

```diff
@@ -141,7 +142,7 @@ const findCollectionOptions = {
 	],
 };
 
-export const findCollection = (
-	collectionId: string,
-): Promise<types.DefinitelyHas<types.Collection, 'community' | 'attributions'>> =>
-	Collection.findOne({ where: { id: collectionId }, ...findCollectionOptions });
+export const findCollection = (collectionId: string) =>
+	Collection.findOne({ where: { id: collectionId }, ...findCollectionOptions }) as Promise<
+		types.DefinitelyHas<Collection, 'community' | 'attributions'>
+	>;
```

Comment:
Not sure why it is typed like this other than convenience, but I decided to leave it to break as little as possible.